src/components/profile/ViewOrUploadAvatar.js

```diff
@@ -109,21 +111,31 @@ const getStylesFromProps = ({ theme }) => ({
     position: 'absolute',
     top: 1,
     right: 1,
-    marginRight: getDesignRelativeWidth(-30),
+    marginTop: 18,
```

Comment:
basically margins and paddings are usually multiples of 8.
use from theme.sizes.default=8 and theme.sizes.defaultDouble=16 defaultQuadruple=32